service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1254,31 +1267,89 @@ public void deleteOrder(Long id) {
     }
 
     @Override
-    public FondyOrderResponse processOrderFondyClientForIF(OrderFondyClientDto dto) throws Exception {
-        Order order = orderRepository.findById(dto.getOrderId()).orElseThrow();
+    public FondyOrderResponse processOrderFondyClientForIF(OrderFondyClientDto dto, String uuid) throws Exception {
+        Order order = orderRepository.findById(dto.getOrderId())
```

Comment:
Order order = orderRepository.findById(dto.getOrderId())
            .orElseThrow(() -> new OrderNotFoundException(ORDER_WITH_CURRENT_ID_DOES_NOT_EXIST));

this can be extracted into a separate method as looks like  it is used not only here